lib/core/macros.nim

```diff
@@ -1344,6 +1348,93 @@ proc copy*(node: NimNode): NimNode {.compileTime.} =
   ## An alias for copyNimTree().
   return node.copyNimTree()
 
+type GenAstOpt* = enum
+  kNoExposeLocalInjects,
```

Comment:
I am sorry, but you have to find a better name for that enum, this is just cryptic.

Reply:
fixed in latest commit